sdk/synapse/Azure.Analytics.Synapse.Artifacts/src/Generated/Models/NotebookCell.Serialization.cs

```diff
@@ -33,15 +33,8 @@ void IUtf8JsonSerializable.Write(Utf8JsonWriter writer)
             writer.WriteEndArray();
             if (Optional.IsDefined(Attachments))
             {
-                if (Attachments != null)
```

Comment:
Property is not nullable in spec, however it is checked via `Optional.IsDefined`